upgrade/v110to120/upgrade.go

```diff
@@ -52,22 +52,22 @@ func upgradeInstanceManagers(namespace string, lhClient *lhclientset.Clientset,
 	if err != nil {
 		return err
 	}
-	for _, imPod := range imPodList {
+	for i, imPod := range imPodList {
```

Comment:
Similar here:
```suggestion
for i := range imPodList.Items {
	imPod := &imPodList.Items[i]
```

Reply:
Here when I'm trying to add the `Items` field to `imPodList` .... I'm getting an error saying `Pod has no field or method Items`